benchflow-experiment-manager/application/src/main/java/cloud/benchflow/experimentmanager/scheduler/ExperimentTaskScheduler.java

```diff
@@ -319,24 +275,54 @@ private void handleReExecuteTrial(String trialID) {
 
       ReExecuteTrialTask reExecuteTrialTask = new ReExecuteTrialTask(trialID);
 
-      Future<TrialStatus> future = experimentTaskExecutorService.submit(reExecuteTrialTask);
+      Future<FabanStatus> future = experimentTaskExecutorService.submit(reExecuteTrialTask);
 
       String experimentID = BenchFlowConstants.getExperimentIDFromTrialID(trialID);
       experimentTasks.put(experimentID, future);
 
       // TODO - change this when faban interaction changes to non-polling
```

Comment:
Ok, the activity is tracked in the following issue: https://github.com/benchflow/benchflow/issues/57